pokemongo_bot/inventory.py

```diff
@@ -319,6 +319,85 @@ def has_space_for_loot(cls):
         max_number_of_items_looted_at_stop = 5
         return cls.get_space_left() >= max_number_of_items_looted_at_stop
 
+class AppliedItem(object):
+    """
+    Representation of an applied item, like incense.
+    """
+    def __init__(self, item_id, expire_ms, applied_ms):
+        """
+        Representation of an applied item
+        :param item_id: ID of the item
+        :type item_id: int
+        :param expire_ms: expire in ms
+        :type expire_ms: in
+        :param applied_ms: applied at
+        :type applied_ms: int
+        :return: An applied item
+        :rtype: AppliedItemItem
+        """
+        self.id = item_id
+        self.name = Items.name_for(self.id)
+        self.applied_ms = applied_ms
+        self.expire_ms = expire_ms
+
+    def refresh(self,inventory):
+        self.retrieve_data(inventory)
+
+    def parse(self, item):
+        if not item:
+            item = {}
+
+        self.id = item.get('id', 0)
+        self.name = Items.name_for(self.id)
+        self.expire_ms = item.get('expire_ms', 0)
+        self.applied_ms = item.get('applied_ms', 0)
+
+    def retrieve_data(self, inventory):
+        ret = {}
+        for item in inventory:
+            data = item['inventory_item_data']
+            if self.TYPE in data:
+                item = data[self.TYPE]
+                ret = item
+                self.parse(item)
+
+        return ret
+
+    def __str__(self):
+        return self.name
+
+
+class AppliedItems(_BaseInventoryComponent):
```

Comment:
AppliedItem(s) is a seperate class in pgoapi, it are items that are in use by the player with a countdown. Like a LuckyEgg and a Incense. 
So it isn't the same as a incense in your inventory, it's a item that is being used.
my knowledge of Python is minimal (I learn as I go) so I just created two classes with Item and Items as example.